src/components/AddressSearch/index.tsx

```diff
@@ -271,6 +271,7 @@ function AddressSearch(
     };
 
     const renderHeaderComponent = () => (
+        // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
this fragment isn't useless ... broken lint rule?

Reply:
It is a bit useless:
```ts
    const renderHeaderComponent = () =>
        predefinedPlaces.length > 0 && (
            <>
                {/* This will show current location button in list if there are some recent destinations */}
                {shouldShowCurrentLocationButton && (
                    <CurrentLocationButton
                        onPress={getCurrentLocation}
                        isDisabled={isOffline}
                    />
                )}
                {!value && <Text style={[styles.textLabel, styles.colorMuted, styles.pv2, styles.ph3, styles.overflowAuto]}>{translate('common.recentDestinations')}</Text>}
            </>
        );
```

The problem lies in `GooglePlacesAutocomplete` component, where it accepts:
```ts
  renderHeaderComponent?: () => JSX.Element | React.ComponentType<{}>;
```

However without `<></>` this function results in `false | Element`

Reply:
That's why I decided to suppress the error